### ISSUE TITLE
[Accessibility] Bar chart/ Pie chart, wont able to navigate using keyboard when NVDA or JAWS is ON

### ISSUE THREAD

**Issue by MdAmeerMind** [highcharts/highcharts#10349]
Expected behaviour
I should navigate through column for Bar chart/Pie Chart when JAWS or NDVA is ON  

Actual behaviour
Unable to navigate on option when Narrator is ON

Version:
Highchart: https://www.npmjs.com/package/highcharts

Live demo with steps to reproduce
Bar Chart: http://jsfiddle.net/2fkr45od/
Pie Chart: http://jsfiddle.net/80fv1os4/

Steps:
Use keyboard to focus and try to navigate using keyboard.

**Comment by sebastianbochan** [comment]
Hi @MdAmeerMind,
Thanks for reporting! Our accessibility expert is currently out of the office (presenting at CSUN), but will look into this when he's back.

**Comment by oysteinmoseng** [comment]
Many screen readers use keyboard shortcuts when they are running, and this can stop the charts from being navigated. For some screen readers you can change this behavior on the fly (disable JAWS virtual cursor, disable Narrator's Scan mode), or pass through certain keys using specific keyboard commands. Unfortunately we cannot change this behavior without losing other screen reader functionality. This is a limitation of current screen readers.

Note that even when the Highcharts keyboard navigation is not working, screen reader users will still be able to navigate the data points of the chart using the virtual cursor. The tooltips will however not be highlighted while navigating the points.

Closing this as it should not be a bug on our end, but feel free to reopen if there are further issues or questions.
